content/shell/android/BUILD.gn

```diff
@@ -28,6 +28,7 @@ shared_library("libcontent_shell_content_view") {
     "//content/shell:pak",
     "//media",
     "//skia",
+    "//starboard($starboard_toolchain)",
```

Comment:
DNS

Reply:
Is this needed for now ? We can `TODO(b/123:)` if needed for follow up

Reply:
Not needed, just to show the media team how to do it

Reply:
Should we comment this in the bug, so we know how to build Starboard in Android?